virtualization/windowscontainers/manage-docker/optimize-windows-dockerfile.md

```diff
@@ -224,7 +224,7 @@ afba1a3def0a        38 seconds ago       cmd /S /C mkdir test-4   42.46 MB
 The docker file has now been slightly modified. Notice that the third `RUN` instruction has changed. When Docker build is run against this Dockerfile, the first three instructions, which are identical to those in the last example, use the cached image layers. However, because the changed `RUN` instruction has not been cached, a new layer is created for itself and all subsequent instructions.
 
 ```
-FROM windowsservercore
+FROM mcr.microsoft.com/windows/servercore:ltsc
```

Comment:
minor nit: `ltsc2019`

Reply:
Gah!  There is a merge conflict, and the GitHub merge resolver does not work in the build of Windows I am running, and I have the attention span of a three year old.  If anyone can make this clean - I will merge it in.